core/os/os2/process.odin

```diff
@@ -2,36 +2,42 @@ package os2
 
```

Comment:
I wouldn't worry too much about this part because it needs a designing properly. All of this was dummy placeholder code.

Reply:
Do you want me to just delete the implementation so people don't rely on it?  I could just roll the nuts and bolts into @flysand7's `process` API that he is working on.

Reply:
Good question. Maybe start working with @flysand7 on getting the two ideas in there.

Reply:
I just reverted all of the process changes.  No sense adding function to an API that won't be around in the future.  Flysand is better at API design than me anyway. =]